vendor

```diff
@@ -1 +1 @@
-Subproject commit 2a0c7ade84641cd9de47d9aa8120caad97c4ef79
+Subproject commit ec29c24cf607f8cc35c907b0d0d0d04fe9d2b6b8
```

Comment:
Submodules are confusing. What your PR now says is to downgrade to ec29c24cf607f8cc35c907b0d0d0d04fe9d2b6b8 which was the previous pointer for the submodule. 
I think if you do `cd vendor; git pull origin master; cd ..; git commit -a -m "current vendor submodule"` it should correct itself. If not, ping me on IRC and I'll help.